src/AzureRenderManager/Microsoft.Identity.Web/Client/MsalUiRequiredExceptionFilterAttribute.cs

```diff
@@ -0,0 +1,97 @@
+using Microsoft.AspNetCore.Authentication;
```

Comment:
Is this class needed or are you just importing the whole lot?

Reply:
Removed along with some others.